src/com/facebook/buck/android/AndroidPrebuiltAarDescription.java

```diff
@@ -106,7 +106,11 @@ public PrebuiltJar apply(AndroidPrebuiltAar input) {
             pathResolver,
             ImmutableSortedSet.<BuildRule>copyOf(javaDeps)));
     AndroidResource androidResource = buildRuleResolver.addToIndex(
-        createAndroidResource(unzipAar, params, pathResolver));
+        createAndroidResource(
+            unzipAar,
+            params,
+            pathResolver,
+            buildRuleResolver.getAllRules(args.deps.get())));
```

Comment:
Should this only be `android_resource` deps like we filter out `javaDeps` above?